docs/tutorials/simulations/MMsbm.ipynb

```diff
@@ -0,0 +1,170 @@
+    "\n",
+    "One drawback of utilizing SBM is that each node must be assigned exclusevily to a single community. Therefore each row of $Z$ contains $1$ in the $j^{th}$ column indicating the community to which that node belongs to and $0$ for all the other entries. \n",
+    "\n",
+    "The Mixed Membership Stochaistic Block Model(MMSBM) addresses this issue by adding some variability in the community assignment process to each node. Indeed, given a network with $k$ communities and a connectivity matrix $B\\in R^{kxk}$ each node $p$ is assigned a membership vector $\\vec{\\pi}_p$. As aforementioned, the $i^{th}$ entry of this vector indicates the probability of node $p$ pertaining to the $i^{th}$ community when interactong with another node. Differently from SBM, a node is allowed to pertain to multiple communities. This is achieved by estimating how often a node act as belonging to a community with respect to another when forming connections with other nodes. For example consider:\n",
+    "\n",
+    "\\begin{align*}\n",
+    "\\vec{\\pi}_p &= \\begin{bmatrix} 1 & 0 & 0 & 0 \\end{bmatrix}\\\\\n",
+    "\\vec{\\pi}_q &= \\begin{bmatrix} 0.25 & 0.25 & 0.25 & 0.25 \\end{bmatrix}\\\\\n",
+    "\\end{align*}\n",
+    "\n",
+    "According to $\\vec{\\pi}_p$, node $p$ will always behave as pertaining to the first community when interacting with other nodes, while node $q$ will interact 25% of the time as if belonging to community 1, 25% of the time as if belonging to community 2, etc.\n",
+    "\n",
+    "In MMSBM, this memership assignment is performed by sampling each membership vector $\\vec{\\pi}_p$ from a Dirichlet distribution with concentration parameter $\\alpha$ so that by modifying $\\alpha$ we can control the shape of the Dirichlet distribution and thus inderectly the assigned membership vectors. For example, when one of the entries in alpha is much greater than the others, the majority of the nodes will pertain to the community associated with that entry. If all entries of $\\alpha$ are identical to each other then values of the concentration parameter closer to $0$ will cause most of the nodes to belong in only one community similarly to an SBM. Always maintaining all entries identical, as we increse the values in $\\alpha$ more nodes are allowed to pertain to multiple communities, therefore the underlying block structure is progressively lost.\n",
```

Comment:
I think in the 2nd sentence (For example, when one of the entries...) you want the \alpha symbol as well